test/conservation_checker_tests.jl

```diff
@@ -82,17 +84,29 @@ for FT in (Float32, Float64)
         )
 
         # set non-zero radiation and precipitation
-        F_r = cf.radiative_energy_flux_toa
+        F_r = 200 .* CC.Fields.ones(space)
         P = cf.P_liq
         Δt = float(cs.Δt_cpl)
 
+        volume = integral(ones(space))
+        # 0.5 is the area fraction
```

Comment:
which area fraction? If it's ice (which has area fraction 0.5), let's use `get_field(ice, Val(:area_fraction))` instead

Reply:
Added the get_field